pkg/cmd/pulumi/state/state_unprotect.go

```diff
@@ -40,10 +40,12 @@ func newStateUnprotectCommand() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "unprotect [resource URN]",
 		Short: "Unprotect resources in a stack's state",
-		Long: `Unprotect resource in a stack's state
+		Long: `Unprotect resource in a stack's state, allowing deletion
```

Comment:
```suggestion
		Long: `Unprotect resources in a stack's state
```
I think the `allowing deletion` bit is covered in the next line, so I'd just fix the typo here.